src/terms/anyonic.jl

```diff
@@ -82,7 +82,9 @@ struct TermAnyonic{T <: Real, Tρ, TA} <: Term
     ρref::Tρ
     Aref::TA
 end
+
 function TermAnyonic(basis::PlaneWaveBasis{T}, hbar, β) where {T}
+    @assert isa(basis.architecture, CPU) "Anyonic term is not implement for GPU yet."
```

Comment:
We don't really do asserts with a message. In such a case we would just have the assertion and a comment in the code. Alternatively use the `error( )` function if you think that's cleaner.

The argument for a simple `@assert` here is (a) Anyonic models are a pretty specific use case and (b) when this  happens, the user would anyway need to open the code to fix it.

Reply:
Makes sense, I have changed it to a simple comment.